pycbc/workflow/segment.py

```diff
@@ -1428,12 +1428,16 @@ def get_segments_file(workflow, name, option_name, out_dir):
         server = cp.get("workflow-segments",
                                  "segments-database-url")
 
+    source = "any"
+    if cp.has_option("workflow-segments", "segments-source"):
+        source = cp.get("workflow-segments", "segments-source")
+
     segs = {}
     for ifo in workflow.ifos:
         flag_str = cp.get_opt_tags("workflow-segments", option_name, [ifo])
         key = ifo + ':' + name
         segs[key] = query_str(ifo, flag_str, start, end,
-                              server=server,
+                              server=server, source=source,
```

Comment:
in the definition of `query_str()` source comes before server, I suppose it works regardless of the order of kwargs but easier to read if they come in the same order.

Reply:
Changed as suggested